.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,27 +36,27 @@ Closes #
 
 -->
 
-## Checklist
+### Documentation
 
-<!-- Please check (put an "x" inside the "[ ]") the applicable items below to make sure your PR is ready to be reviewed. -->
+<!-- Please check (put an "x" inside the "[ ]") the applicable items below to indicate which documentation has been updated. -->
 
-- [ ] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
-- [ ] Make sure to add/update documentation regarding your changes
+- [ ] Add or update documentation reflecting your changes
 - [ ] If you are deprecating/removing a feature, make sure to update
       [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)
 
-#### Maintainers
+## Checklist for Maintainers
 
 - [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
-- [ ] Make sure this PR contains **one** of the labels below.
-
-`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`
-
-<!--
-
-Everybody: Please submit all PRs to the `next` branch unless they are specific to the current release. Storybook maintainers cherry-pick bug and documentation fixes into the `main` branch as part of the release process, so you shouldn't need to worry about this. For additional guidance: https://storybook.js.org/docs/react/contribute/how-to-contribute
-
--->
+- [ ] Make sure this PR contains **one** of the labels below:
+- `cleanup`: Minor cleanup style change. Will not show up in release changelog.
+- `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
+- `documentation`: Documentation **only** changes. Will not show up in release changelog.
+- `maintenance`: User-facing maintenance tasks.
+- `dependencies`: Upgrading (sometimes downgrading) dependencies.
+- `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
+- `feature request`: Introducing a new feature.
+- `bug`: Internal changes that fixes incorrect behavior.
+- `other`: Changes that don't fit in the above categories.
```

Comment:
Minor thing, but these should probably be indented:

```suggestion
- [ ] Make sure this PR contains **one** of the labels below:
   - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
   - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
   - `documentation`: Documentation **only** changes. Will not show up in release changelog.
   - `maintenance`: User-facing maintenance tasks.
   - `dependencies`: Upgrading (sometimes downgrading) dependencies.
   - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
   - `feature request`: Introducing a new feature.
   - `bug`: Internal changes that fixes incorrect behavior.
   - `other`: Changes that don't fit in the above categories.
```


Or, we could hide these in a `details`? They'll still be a visible list when editing and they'll collapse by default when viewing PRs.

```suggestion
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `feature request`: Introducing a new feature.
     - `bug`: Internal changes that fixes incorrect behavior.
     - `other`: Changes that don't fit in the above categories.
   </details>
```

You may need to tweak the line breaks to get that to work, or it may not work at all. 🤷